src/Common/CustomLogStyle.php

```diff
@@ -0,0 +1,154 @@
+<?php
+namespace Robo\Common;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\OutputStyle;
+
+/**
+ * Custom Log Styler. Styles may be overridden via constructor parameters.
+ */
+class CustomLogStyle implements LogStyleInterface
```

Comment:
Can be renamed to ConsoleLogStyle 
as it is not custom but common class (used by Robo) and available for cli usage